code/game/objects/items/storage/belt.dm

```diff
@@ -575,7 +575,10 @@
 		/obj/item/clothing/gloves,
 		/obj/item/melee/flyswatter,
 		/obj/item/paint/paint_remover,
-		/obj/item/assembly/mousetrap
+		/obj/item/assembly/mousetrap,
+		/obj/item/screwdriver,
+		/obj/item/stack/cable_coil,
+		/obj/item/restraints/handcuffs/cable

```

Comment:
why are you buffing belts in a cargo crate addition

Reply:
I can remove the cuffs and just keep the cables, it seemed appropriate if I was to do one, I should do the other. Light fixtures were the main focus of it, yes.

I did it in this because I didnt feel like it was necessary to do two separate PR's for it but I'll address it when I get home.

Reply:
Cable cuffs fit tho, The Owl and all